tests/client_no_stream_test.go

```diff
@@ -39,3 +40,11 @@ func TestStatsNoStream(t *testing.T) {
 		t.Error("No stream block should result in StreamZoneSync = `nil`")
 	}
 }
+
+func getTestEndpoint() string {
```

Comment:
consider adding a helper file with the name like `env_variables.go` where you would have functions that get the values from the env variables.

the comment applies to the other similar functions

Reply:
Has to have `_test.go` suffix to be picked up by `go test`. Calling it `env_var_helpers_test.go` for now.